frontend/src/components/Main.js

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect} from 'react'
+import { useSelector, useDispatch, batch } from 'react-redux'
+import { useHistory } from 'react-router-dom'
+
+import { API_URL } from '../reusable/urls'
+
+import thoughts from '../reducers/thoughts'
+import user from '../reducers/user'
+
+
+const Main = () => {
+    const accessToken = useSelector(store => store.user.accessToken)
+
+    const dispatch = useDispatch()
+    const history = useHistory()
+
+    const logout = () => {
+        batch(() => {
+            dispatch(user.actions.setUsername(null))
+            dispatch(user.actions.setAccessToken(null))
+        })
+    }
+
+    useEffect(() => {
+        if (!accessToken) {
+            history.push('/login')
+        }
+    }, [accessToken, history]) 
+
+    useEffect(() => {
+        const options = {
+            method: 'GET',
+            headers: {
+                Authorization: accessToken //added quotations to accessToken
+            }
+        }
+
+        fetch(API_URL('thoughts'), options) 
+            .then(res => res.json())
+            .then(data => {
+                if (data.success) {
+                    batch(() => {
+                        dispatch(thoughts.actions.setThoughts(data.thoughts))
+                        dispatch(thoughts.actions.setErrors(null))
+                    })
+                } else {
+                    dispatch(thoughts.actions.setErrors(data))
+                }
+            })
+    }, [accessToken, dispatch])
+
+    return (
+        <div className="main-wrapper">
+            <iframe className="happy-thoughts-site" width="100%" height="100%" src="https://happy-thoughts-app-estefania.netlify.app/" title="Happy Thoughts site"></iframe>
```

Comment:
This container for the Happy Thoughts website looks too small, maybe try to make it larger ✨🐱‍🏍

![image](https://user-images.githubusercontent.com/73911558/120671029-457c2d00-c491-11eb-9ad2-3823dc3da851.png)

Reply:
Maybe just use the HT project and add a header with a logout button to it instead container?